lib/skate_web/channels/vehicles_search_channel.ex

```diff
@@ -0,0 +1,76 @@
+defmodule SkateWeb.VehiclesSearchChannel do
+  use SkateWeb, :channel
+  use SkateWeb.AuthenticatedChannel
+  require Logger
+
+  alias Realtime.Server
+  alias Util.Duration
+
+  @impl SkateWeb.AuthenticatedChannel
+  def join_authenticated(
+        "vehicles_search:",
+        %{"limit" => limit, "text" => text, "property" => property},
+        socket
+      ) do
+    username_from_socket! =
+      Application.get_env(
+        :skate,
+        :username_from_socket!,
+        &SkateWeb.AuthManager.username_from_socket!/1
+      )
+
+    username = username_from_socket!.(socket)
+    %{id: user_id} = Guardian.Phoenix.Socket.current_resource(socket)
+
+    subscribe_args = %{
+      property: String.to_existing_atom(property),
+      limit: limit,
+      text: text
+    }
+
+    subscribe_args =
+      Map.put(
+        subscribe_args,
+        :include_logged_out_vehicles,
+        Skate.Settings.User.is_in_test_group(user_id, "search-logged-out-vehicles")
+      )
+
+    Logger.info(fn ->
+      "User=#{username} searched for property=#{subscribe_args.property}, text=#{subscribe_args.text} limit=#{subscribe_args.limit}"
+    end)
```

Comment:
**suggestion (non-blocking): it might be good to have a clearer delineation between this logging and the logging in `VehiclesChannel` besides just the presence / absence of the `limit` field.

Reply:
Good point. I'm thinking of added something like "limited" to the topic name too for further differentiation of these channels.

Reply:
Made both changes in https://github.com/mbta/skate/pull/2124/commits/e320a2364b3e5377b661a6a6d222c0a72fd53b3b